backend/Domain/Models/User.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.Collections.Generic;
+using System.ComponentModel;
+
+namespace OpenData.Domain.Models
+{
+    public enum UserType: byte
+    {
+        [Description("standard")]
+        Standard,
+        [Description("admin")]
+        Admin
+    }
+
+    public class User
+	{
+		public string Mail { get; set; }
+		public string Password { get; set; }
+		public string PasswordSalt { get; set; }
+        public UserType UserType { get; set; } = UserType.Standard;
```

Comment:
Fix tab

Reply:
<img width="577" alt="image" src="https://user-images.githubusercontent.com/31246008/73680053-dedbd900-46bb-11ea-99eb-0387102b3c9b.png">

Reply:
Very weird, cannot reproduce the tab error.